lib/individual/health.dart

```diff
@@ -14,169 +15,249 @@ class _HealthPageState extends State<HealthPage> {
-                                      borderRadius: BorderRadius.circular(80.r),
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.center,
+                                  children: [
+                                    SizedBox(
+                                      width: 40.w,
                                     ),
-                                    focusedBorder: OutlineInputBorder(
-                                      borderRadius: BorderRadius.circular(80.r),
-                                      borderSide:
-                                          BorderSide(color: const Color.fromARGB(255, 137, 184, 189), width: 2.w),
+                                    Text(
+                                      "Date",
+                                      style: TextStyle(
+                                          fontWeight: FontWeight.w500,
+                                          fontSize: 20.sp),
                                     ),
-                                    fillColor: const Color.fromARGB(255, 186, 231, 235),
-                                    filled: true,
-                                    suffixIcon: const Icon(
-                                      Icons.arrow_drop_down,
-                                      color: Colors.grey,
-                                      size: 40,
+                                    // SizedBox(
+                                    //   width: 130.w,
+                                    // ),
+                                    // Text(
+                                    //   "Time",
+                                    //   style: TextStyle(
+                                    //       fontWeight: FontWeight.w500,
+                                    //       fontSize: 20.sp),
+                                    // ),
+                                    SizedBox(
+                                      width: 40.w,
                                     ),
-                                  ),
+                                    // SizedBox(
+                                    //   height: 10.h,
+                                    // )
+                                    //
+                                  ],
                                 ),
-                              ),
-                            ],
-                          )
-                        ]),
-                  ),
-                ],
-                // ]
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.center,
+                                  children: [
+                                    Padding(
+                                      padding:
+                                          EdgeInsets.fromLTRB(15.w, 0, 15.w, 0),
+                                    ),
+                                    Expanded(
+                                      child: TextField(
```

Comment:
I think it will be better to use **DatePicker** to allow the user to select a Date. :thinking: 
What do you think? I believe it will improve the overall UX of the page.

Reference - 
> https://stackoverflow.com/questions/52727535/what-is-the-correct-way-to-add-date-picker-in-flutter-app
> https://api.flutter.dev/flutter/material/showDatePicker.html